### PR TITLE
feat(locale): Added ru locale

### PR DESCRIPTION
**Pull request by engineerklimov**
Added Russian (ru) locale.

### CODE REVIEW

**Comment by engineerklimov** [comment]
?

**Comment by straker** [comment]
Thanks for the pull request! We're working on getting our tests passing and then I can review this and get it merged.

**Review by straker — COMMENTED** [review]
Thanks again for the translation. Just have a small comment.

File: locales/ru.json [hunk 1]
```diff
@@ -0,0 +1,1123 @@
...
+      "description": "Убедитесь, что кнопки имеют различимый текст",
+      "help": "Кнопки должны иметь различимый текст"
+    },
+    "bypass": {
+      "description": "Убедитесь, что на каждой странице есть хотя бы один механизм, позволяющий пользователю обходить навигацию и переходить непосредственно к содержимому",
+      "help": "На странице должен быть способ обойти повторяющиеся блоки"
+    },
+    "color-contrast-enhanced": {
+      "description": "Убедитесь, что контраст между цветами переднего и заднего плана соответствует пороговым значениям контрастности WCAG 2 AAA",
+      "help": "Элементы должны соответствовать пороговым значениям контрастности цвета"
```
[the remark this answers is not in the document]
Comment (engineerklimov) [reply locales/ru.json]: Thanks for your comment, i have done adjustment.

**Review by straker — APPROVED** [review]
Awesome!

**Pull request merged**